forth_src/defer.fs

```diff
@@ -5,3 +5,10 @@ does> @ execute ;
 : is state @ if
 postpone ['] postpone defer!
 else ' defer! then ; immediate
+: hide ( word -- )
```

Comment:
I have not been doing it proper myself but maybe `: hide ("name" -- )` would clearer convey the usage. Also, this word should probably be added to the manual.